src/types/types.tsx

```diff
@@ -0,0 +1,19 @@
+import type { RootState } from '../app/store';
+
+export interface IUser {
```

Comment:
Prefixing an interface with "I" in TypeScript generally a bad practice. So I would suggest to avoid this. https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names
https://stackoverflow.com/questions/31876947/confused-about-the-interface-and-class-coding-guidelines-for-typescript/41967120#41967120